source/geant4.f90

```diff
@@ -0,0 +1,145 @@
+module geant4
+
+  use floatPrecision
+  use crcoall
+  use iso_c_binding, only: C_NULL_CHAR
+  use numerical_constants
+
+  implicit none
+
+  real(kind=fPrec) :: g4_recut
+  real(kind=fPrec) :: g4_aecut
+  real(kind=fPrec) :: g4_rcut
+  real(kind=fPrec) :: g4_rangecut
+  integer :: g4_physics
+  integer :: g4_keep
+
+  character(len=64) :: phys_str
+
+  logical :: g4_enabled
+  logical :: g4_debug
+  logical :: g4_keep_stable
+
+contains
+
+subroutine geant4_fortran_init()
```

Comment:
Any reason why this isn't initialised in the module header?

Reply:
tis from a time before modules existed. Updated now.

Reply:
Ah, right. Great!